rfcs/20230606-topk.md

```diff
@@ -0,0 +1,88 @@
+# RFC: Add TopK Op to StableHLO
+
+Status: Draft<br/>
+Initial version: 2023-06-06<br/>
+Last updated: 2023-06-06
+
+## Summary
+
+We propose to add a new op: `stablehlo.topk`, which returns the first `k`
+elements of a tensor and their indices, based on a user-provided comparator. The
+goal of the new op is to bridge a gap between existing and emerging uses of TopK
+in models (especially transformers and LLMs, e.g.: [^1], [^2]), as well as in
+the IR lowering chain.
+
+## Background and Motivation
+
+The TopK operation is ubiquitous in both ML frameworks (e.g.,
```

Comment:
s/both/many ?

Reply:
The 'both' refers to (1) ML frameworks and (2) compiler IRs. This is a bit clumsy, so I changed it like you suggested. Thanks.